cmd/stork/stork.go

```diff
@@ -590,6 +590,13 @@ func runStork(mgr manager.Manager, ctx context.Context, d volume.Driver, recorde
 		}
 	}
 
+	if c.Bool("webhook-controller") {
+		log.Infof("Creating mutating webhook after leader election")
+		if err := webhookadmission.CreateMutateWebhookRuntime(); err != nil {
+			log.Fatalf("Error creating webhook: %v", err)
+		}
+	}
+
```

Comment:
Not related to review, but couldn't understand the context of the case when the driver would be nil and stork would be run with `--webhook-controller=true` flag.

Reply:
Theoretically No, but because the underlying function is not using driver as any parameter, we can only put under the condition `webhook-controller`.